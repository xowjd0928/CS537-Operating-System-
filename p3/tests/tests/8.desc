Commands in comments should not be run. Score: 1