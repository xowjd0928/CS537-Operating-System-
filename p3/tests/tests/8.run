../solution/wsh <tests/8.wsh
