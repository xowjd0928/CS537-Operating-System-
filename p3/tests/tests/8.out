wsh> a
wsh> wsh> c
wsh> wsh> e
wsh> 